src/main/java/org/commcare/core/process/CommCareInstanceInitializer.java

```diff
@@ -293,12 +294,33 @@ public String getVersionString() {
         return "CommCare Version: " + mPlatform.getMajorVersion() + "." + mPlatform.getMinorVersion();
     }
 
-    public static class FixtureInitializationException extends RuntimeException {
+    public static class FixtureInitializationException extends RuntimeException implements WithHtmlMessage {
+
+        private String htmlMessage;
+
+        private static String getName(String lookupReference) {
+            int lastColonIndex = lookupReference.lastIndexOf(':');
+            if (lastColonIndex == -1) {
+                return lookupReference;
+            }
+            return "%s (%s)".formatted(lookupReference.substring(lastColonIndex + 1), lookupReference);
+        }
 
         public FixtureInitializationException(String lookupReference) {
             super(Localization.getWithDefault("lookup.table.missing.error",
                     new String[]{lookupReference},
-                    "Unable to find lookup table: " + lookupReference));
+                    """
+                    Unable to load lookup table: %s
+                    Please go to Home → Settings → 'Clear user data' and then 'Sync user data'.
+                    If the issue persists please contact CommCare Support.
+                    """.formatted(getName(lookupReference))));
```

Comment:
Think this message is speicif to formplayer and needs to be overwritten on `formplayer` side as mobile users don't have the same navigation path and should never do `Clear user data`. We should instead have a generic message here that just asks user to sync again without the exact UI steps.